3rdparty/cozytouch/client/CozyTouchApiClient.class.php

```diff
@@ -45,12 +45,12 @@ public function authenticate()
 	public function getToken() {
 		$post_data = array(
 				'grant_type' => 'password',
-				'username' => $this->userId,
+				'username' => 'GA-PRIVATEPERSON/'.$this->userId,
```

Comment:
@jeremiedmns could you explain what this `GA-PRIVATEPERSON/` addition means? I work on the CozyTouch (overkiz) integration for Home Assistant. :)

Reply:
Hello @iMicknl 
GA-PRIVATEPERSON is the store of cozytouch accounts. If you don’t add it with new url, you could have troubles on login.
You can use same basic with new url for homeassistant integration.
Contact me if you have any question.

Reply:
Thanks @jeremiedmns! I tested it without adding `GA-PRIVATEPERSON/`, and that seem to work. But I can imagine that adding this prefix will be more future ready.

Reply:
@jeremiedmns where did you got this knowledge? (You can also reach out privately to me on Discord, iMick#1903).

I wonder if this is required, or only in scenarios where business accounts / private persons share the same email address.